execution/executor/tests/db_bootstrapper_test.rs

```diff
@@ -238,6 +238,7 @@ fn restore_state_to_db(
 }
 
 #[test]
+/// Pre genesis state should be empty. This test makes sure that if there is anything in the DB before genesis, an error will be returned.
 fn test_pre_genesis() {
```

Comment:
I would just remove the concept of "anything in the DB before genesis" -- hence delete this test.

Your test is working only because other parts of the code still assumes PRE_GENESIS_VERSION is before 0.

Reply:
Done.